lib/pharos/scripts/configure-etcd.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -e
+
+if [ ! -e /etc/kubernetes/kubelet.conf ]; then
+  mkdir -p /etc/systemd/system/kubelet.service.d
+  cat <<EOF >/etc/systemd/system/kubelet.service.d/5-pharos-etcd.conf
```

Comment:
BTW: calling these `5-pharos-*.conf` is super confusing, because that implies that they would sort before the `10-kubeadm.conf`. Instead, they actually sort after, so they should really be called something like `50-pharos-*.conf`:

```
$ ls -l /etc/systemd/system/kubelet.service.d/
total 8
-rw-r--r-- 1 root root 937 Mar 27 15:50 10-kubeadm.conf
-rw-r--r-- 1 root root 243 Apr 13 14:09 5-pharos.conf
```

Reply:
And renaming these to `05-pharos.conf` would probably break it, they need to come after the `10-kubeadm.conf`.

Reply:
Created an issue about this: https://github.com/kontena/pharos-cluster/issues/242